### PR TITLE
Media Lib: remove noticon from source picker

### PR DESCRIPTION
**Pull request by drw158**
This removes the chevron noticon that was being used for the source picker button.

Notice that the icon is darker now, and does not change rotation. I did not change the icon in the add image button, but shown in the gifs just for demo purposes.

I tested mobile viewport widths in Chrome. The media library in the post editor uses the same components, so it works just the same there also.

Before

before

After

after

---

I decided to also take care of the placeholder image noticon. I just removed it instead of replacing it with a gridicon. I don't think the image icon in the placeholder adds much value.

before 

before 2

after

after 3

### CODE REVIEW

**Comment by drw158** [comment]
Not sure what the failed check is about.

File: client/my-sites/media-library/style.scss [hunk 1]
```diff
@@ -142,45 +142,14 @@
...
-.media-library__source-button.is-open.is-borderless {
-	&:after {
-		@include noticon( '\f432', 20px );
-	}
-
-	.gridicon {
-		color: $gray-dark;
+	.gridicons-chevron-down {
+		width: 18px;
+		height: 18px;
```
Comment (shaunandrews) [inline client/my-sites/media-library/style.scss]: Why do we need to specify the height/width here?

Comment (drw158) [reply client/my-sites/media-library/style.scss]: If it's not there, then this bit of css overrides the 18px size. I think it's in the button component css:

screen shot 2018-03-06 at 4 24 24 pm

Comment (shaunandrews) [reply client/my-sites/media-library/style.scss]: Oh yea, I've run in to this before.

**Pull request merged**